### ISSUE TITLE
Disabled BUY button 

### ISSUE THREAD

**Issue by roiLeo** [kodadot/nft-gallery#6309]
Tested on https://kodadot.xyz/stmn/gallery/19-2

![Screenshot 2023-06-22 at 14-04-30 NFT Universe 2 KodaDot - NFT Market Explorer](https://github.com/kodadot/nft-gallery/assets/9987732/a955c806-5e1c-4e59-9cbf-4bec56b50dfb)

![Screenshot 2023-06-22 at 14-06-37 NFT Universe 2 KodaDot - NFT Market Explorer](https://github.com/kodadot/nft-gallery/assets/9987732/e0e4351e-3e3a-4a55-a231-3df2ec755de9)

**Comment by prury** [comment]
dude, i just found this one too, rn

**Comment by roiLeo** [comment]
> dude, i just found this one too, rn

gotta go fast

**Comment by daiagi** [comment]
issue caused by 'auto' width on tooltip
```
  <o-tooltip
    v-if="active"
    :append-to-body="appendToBody"
    :multiline="multiline"
    class="neo-tooltip"
    :style="{
      '--font-size': fontSize,
      '--multiline-width': multilineWidth,
      width: fullWidth ? '100%' : 'auto',
    }"
    
    ```
    
from PR:

* https://github.com/kodadot/nft-gallery/pull/6271


@Jarsen136  you must have put it there for a reason, whatever issue you were solving, can it be solved in another way?

**Comment by yangwao** [comment]
> gotta go fast

![image](https://github.com/kodadot/nft-gallery/assets/5887929/579111dc-949c-4c85-be7b-63789460a74e)

https://youtu.be/rJOsjP33nF4?t=92

**Comment by Jarsen136** [comment]
> 
> from PR:
> 
> * [fix: Create drop down overflowing #6271](https://github.com/kodadot/nft-gallery/pull/6271)
> 
> @Jarsen136 you must have put it there for a reason, whatever issue you were solving, can it be solved in another way?

Sorry for missing this comment. I have fixed the style issue on #6412
